doc/userguide/performance/rule-profiling.rst

```diff
@@ -1,6 +1,14 @@
 Rule Profiling
 ==============
 
+Suricata can generate a rules performance report at the end of each session, if
+built with the ``enable-profiling`` option (see configuring :ref:`Rule profiling<rule-and-packet-profiling-settings>`).
+
+Rule profiling can also be enabled by building the engine with ``enable-profiling-rules``
```

Comment:
What is the difference between the two ?

Reply:
Good question. I think that `enable-profiling-rules` won't enable other profiling options, whereas `enable-profiling` will also generate stats for packet profiling. Not sure if there's more that our docs don't mention.

Reply:
Should I try to improve this section to clarify the difference?

Reply:
> Should I try to improve this section to clarify the difference?

I think so